charts/federation-v2/README.md

```diff
@@ -29,24 +29,37 @@ group](https://groups.google.com/forum/#!forum/kubernetes-sig-multicluster).
 
 ## Installing the Chart
 
-First you'll need to create the reserved namespace for registering clusters with the
-cluster registry:
+First you'll need to create the reserved namespace for registering clusters with the cluster registry:
 
 ```bash
 $ kubectl create ns kube-multicluster-public
 ```
 
-Install the chart with the release name `federation-v2`:
+Next, add the chart to your local resitory.
+```bash
+$ helm repo add federation-v2 https://raw.githubusercontent.com/kubernetes-sigs/federation-v2/master/charts/federation-v2
+
+$ helm repo list
+NAME            URL                                        
+federation-v2   https://raw.githubusercontent.com/kubernetes-sigs/federation-v2/master/charts/federation-v2
+```
+
+With the chart installed, availble versions can be viewed.
```

Comment:
Did you mean 'With the repo added' ?

Also, s/availble/available/

Reply:
Yes! Thank you for that catch.